src/frontend/sharedComponents/OfflineAreaCard.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { StyleSheet, View } from "react-native";
+import { LIGHT_GREY } from "../lib/styles";
+import Text from "./Text";
+
+const m = defineMessages({
+  lvlDetail: {
+    id: "sharedComponent.OfflineAreaCard.lvlDetail",
+    defaultMessage: "Level of Detail",
+  },
+  zoomLevel: {
+    id: "sharedComponent.OfflineAreaCard.zoomLevel",
+    defaultMessage: "Zoom Level",
+  },
+});
+
+interface OfflineAreaCardProps {
+  title: string;
+  zoomLevel: number;
+  lvlOfDetail?: string;
```

Comment:
based on https://github.com/digidem/mapeo-mobile/pull/946/files#r843175295, i think that we should generate the appropriate string based on the `zoomLevel` so basically have a map of zoom level ranges to a particular description